docs/builders/observability-builder.md

```diff
@@ -0,0 +1,116 @@
+# Observability Builder
```

Comment:
This has to be included in the overall navigation. You can add a top level module, like builders. But it has to have a doc on the regular builder (we probably already have this content) and a doc on the observability builder that you already have. We should also have a top level into about builders in general, which I can help with.

Reply:
Added the required navigation to `mkdocs.yml` and it will be awesome if you can add an `index.md` for builders in general.